public/style.css

```diff
@@ -469,15 +469,24 @@ nav {
 .qa-search-input {
   width: 100%;
   padding: 10px;
-  /* background-image: url("https://img.icons8.com/ios/24/000000/search--v1.png");
+  padding-left: 22px;
+  background-image: url("https://img.icons8.com/ios/24/000000/search--v1.png");
```

Comment:
Could this also be saved in the images folder for faster retrieval?

Reply:
For sure. On the to-do list!